docs/GettingStarted.md

```diff
@@ -1578,16 +1578,11 @@ The Resque integration uses Resque hooks that wraps the `perform` method.
 To add tracing to a Resque job:
 
 ```ruby
+require 'resque'
 require 'ddtrace'
```

Comment:
Upside: clients don't have to write `require` themselves.
Downside: it might not be transparent what's being `require`d in their application: _"Did ddtrace require 'active_support' or 'active_support/all'?"_.
Also, they might not find any `require` statements by "grepping" their code anymore for a library they use: _"Where is Redis `require`d? I can't find it anywhere..."_.

By the power vested in me, I invoke **@delner** to chime in here.

Reply:
I don't believe this comments blocks this PR, though.

Reply:
Yeap, not blocking at all!